src/read.c

```diff
@@ -4342,41 +4351,49 @@ static avifResult avifDecoderDataFindToneMappedImageItem(const avifDecoderData *
     return AVIF_RESULT_OK;
 }
 
-// Finds a 'tmap' (tone mapped image item) box associated with the given 'colorItem',
+// Finds a 'tmap' (tone mapped image item) box associated with the given colorItem at index 'colorItemIndex',
```

Comment:
Nit: the given colorItem => the color item

Reply:
Done.

Reply:
The change I suggested was not made.

Reply:
sorry i made it in the wrong place. fixed it now.